woocommerce/woocommerce-android
# Issue/6310 inbox UI glitches

Closes: #6310 

### Description
Fixes some UI glitches and adds some improvements to Inbox screen:

- Adds spinner to "Dismiss" action button while the note is dismissing
- Ensure all inbox note actions fit in the screen when they have very long texts. This is done by removing unneeded `Lazyrow` and adding a `Row` that will create a line break for the action text if it doesn't fit in a single line. 
- Improve animation for skeletons in Compose UI

### Testing instructions
- Open Inbox
- Check all actions are properly visible in the list
- Check a spinner is shown (and onClick is disabled) while the note is being dismissed.

### Images/gif
For demo purposes when recording the video, to showcase the skeleton animation, I added a delay of 4s. 

https://user-images.githubusercontent.com/2663464/164729296-65677e13-9294-4531-a4e8-987667168e19.mp4

To showcase the animation change in the skeletons. Here is the before and after. 

**Before**

https://user-images.githubusercontent.com/2663464/164732247-24fbf349-0d94-4412-a54f-7ed35e5ff909.mp4

**After**


https://user-images.githubusercontent.com/2663464/164732010-fcf51b6a-446d-47e8-8ed2-21764167de76.mp4
